### PR TITLE
Native CWPT | Centaur/Vega

### PR DESCRIPTION
**Pull request by DeltaDizzy**

### CODE REVIEW

File: Gamedata/Bluedog_DB/Parts/Centaur/bluedog_Centaur_Avionics.cfg [hunk 1]
```diff
@@ -35,7 +35,8 @@ MODEL
 	bulkheadProfiles = size1, size1p5
 	
 	tags = inon centaur probe core remote control 
-	
+	techtag = centaurD
```
Comment (zorg2044) [inline Gamedata/Bluedog_DB/Parts/Centaur/bluedog_Centaur_Avionics.cfg]: Centuar A?

File: Gamedata/Bluedog_DB/Parts/Centaur/bluedog_Centaur_EngineMountA.cfg [hunk 2]
```diff
@@ -68,7 +68,8 @@ MODEL
 	bulkheadProfiles = size1, size1p5
 
 	tags = centaur inon mounting plate fuel structure
-	
+	techtag = centaurD
```
Comment (zorg2044) [inline Gamedata/Bluedog_DB/Parts/Centaur/bluedog_Centaur_EngineMountA.cfg]: Centaur A?

File: Gamedata/Bluedog_DB/Parts/Centaur/bluedog_Centaur_FairingBase.cfg [hunk 3]
```diff
@@ -42,7 +42,8 @@ MODEL
 	bulkheadProfiles = size1p5
 	
 	tags = centaur inon vega vejur aerodynamic shell fairing 1.875 1875 base
-	
+	techtag = centaurD
```
Comment (zorg2044) [inline Gamedata/Bluedog_DB/Parts/Centaur/bluedog_Centaur_FairingBase.cfg]: centaur A?

File: Gamedata/Bluedog_DB/Parts/Centaur/bluedog_centaur25mAdapterFairing.cfg [hunk 4]
```diff
@@ -36,7 +36,6 @@ MODEL
 	bulkheadProfiles = size1p5, size2
 	
 	tags = centaur inon atlasv atlas5 ?5 ?v muo ?atlas eelv ccb interstage decoupler fairing cover adapter 1.875 2.5
-
```
Comment (zorg2044) [inline Gamedata/Bluedog_DB/Parts/Centaur/bluedog_centaur25mAdapterFairing.cfg]: Techtag missing

File: Gamedata/Bluedog_DB/Parts/Centaur/bluedog_Centaur_FairingBase.cfg (hunk 3 above)
Comment (DeltaDizzy) [reply Gamedata/Bluedog_DB/Parts/Centaur/bluedog_Centaur_FairingBase.cfg]: I thought Kore said he would fix all these but oh well...

File: Gamedata/Bluedog_DB/Parts/Centaur/bluedog_centaur25mAdapterFairing.cfg (hunk 4 above)
Comment (DeltaDizzy) [reply Gamedata/Bluedog_DB/Parts/Centaur/bluedog_centaur25mAdapterFairing.cfg]: Isn't that part deprecated?

**Pull request merged**